### DISCUSSION TITLE
Welcome to MCX discussions

### DISCUSSION THREAD

**Discussion by fangq** [fangq/mcx#167] [Announcements]
Over the past 14 years, we have been using our [mcx-users mailing list](https://groups.google.com/g/mcx-users) to answer questions and provide timely software support to our growing user community. Currently, our mcx-users mailing list has over 320 active subscribers.

We are now testing the Github-integrated "Discussions" feature - a forum-like message board for interactions between users and developers. Because of the integration with Github source code management workflows, we anticipate that this feature will offer new opportunities related to code-related discussions, feature request, collaboration and code contributions. It is not yet meant to be a replacement to our mcx-users mailing list. However, we welcome users to participate discussions in either of the channels, and will closely monitor both channels to offer our help.

We will begin making important announcement in both Discussions and the mailing list and explore new features in these platforms to create new ways of supporting our users.

MCX is open-source and we are proud of the openness of our development and thrilled to hear from our users - whether it is a simple question or an in-depth discussion. **Please be part of our active user community** and share your experience, thoughts, suggestions and contributions with us and we can be certain that this actively maintained software will become an increasingly valuable addition to your research.

**Comment by jgroehl** [comment]
Thanks for your ongoing contributions to the research community via MCX, @fangq! It has become an invaluable part of many research projects I am involved in and the continuous maintenance means that we can use it with a high degree of confidence which can be a rare commodity :)

**Comment by fangq** [comment]
@jgroehl, very glad to hear mcx is a useful tool to your work, and we hope to provide better support to our growing user community.

**Comment by Netaniel2** [comment]
I am sorry if this is very obvious to most - how is it that if this system is astochstic system, if I run a simulation several times I recieve the exact same number of photons detected?

**Reply by fangq** [reply]
@Netaniel2, you need to change the seed to obtain different results. this is why it is called PRNG (pseudo random number generator), https://en.wikipedia.org/wiki/Pseudorandom_number_generator

in some applications, reproducible result could be a desirable feature, even though, it is complicated on the GPU to create reproducible outputs, see my notes in this mailing list thread

https://groups.google.com/g/mcx-users/c/DXJ9C70Sajo/m/TVmGaK0NAQAJ
https://groups.google.com/g/mcx-users/c/E9VgPb4q62M/m/d6G_9Dr7AgAJ

or search more in the forum
https://groups.google.com/g/mcx-users/search?q=reproducible
